Changelog.md

```diff
@@ -2,6 +2,8 @@
 
 ## Unreleased (in master)
 
+- Added subtree view to file listings.
```

Comment:
`subtree` feels like an implementation name, rather than a user one (at least it's not readily recognizable to me as generic term for this anyway). I would think of this more as a "recursive" listing, e.g. as used in `ls`. What do you think? We should change this in the UI as well if/when we change it.